compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -3037,7 +3038,7 @@ fn add_apple_link_args(cmd: &mut dyn Linker, sess: &Session, flavor: LinkerFlavo
             _ => bug!("invalid OS/ABI combination for Apple target: {target_os}, {target_abi}"),
         };
 
-        let (major, minor, patch) = current_apple_deployment_target(&sess.target);
+        let (major, minor, patch) = deployment_target(sess);
```

Comment:
```suggestion
        let (major, minor, patch) = apple::deployment_target(sess);
```
Style nit: import `apple`, not the function, this is specifically an Apple deployment target, not some general deployment target.